chart/epinio/templates/_helpers.tpl

```diff
@@ -0,0 +1,5 @@
+{{- define "registry-url" -}}
```

Comment:
I am probably missing something but what's the purpose of this template versus just using {{ .Values.registryURL }}

Reply:
Oops, currently none. I started lazy and [copied from a place where it had more logic](https://github.com/rancher/charts/blob/dev-v2.6/charts/longhorn/longhorn/1.1.100/templates/_helpers.tpl#L49-L55). Then I removed the logic...

I will simplify this.